myriad-scheduler/src/test/java/org/apache/myriad/scheduler/TestMyriadScheduler.java

```diff
@@ -22,17 +22,18 @@
 import org.apache.hadoop.conf.Configuration;
 import org.apache.hadoop.yarn.conf.YarnConfiguration;
 import org.apache.hadoop.yarn.event.AsyncDispatcher;
-//import org.apache.hadoop.yarn.server.resourcemanager.MockNodes;
 import org.apache.hadoop.yarn.server.resourcemanager.ResourceManager;
-//import org.apache.hadoop.yarn.server.resourcemanager.rmnode.RMNode;
 import org.apache.hadoop.yarn.server.resourcemanager.scheduler.ResourceScheduler;
-//import org.apache.hadoop.yarn.server.resourcemanager.scheduler.event.NodeAddedSchedulerEvent;
-//import org.apache.hadoop.yarn.server.resourcemanager.scheduler.event.NodeRemovedSchedulerEvent;
 import org.apache.hadoop.yarn.server.resourcemanager.scheduler.fair.FairScheduler;
 import org.apache.hadoop.yarn.server.resourcemanager.scheduler.fair.FairSchedulerConfiguration;
-//import org.apache.hadoop.yarn.util.resource.Resources;
 import org.junit.After;
 import org.junit.Before;
+
+//import org.apache.hadoop.yarn.server.resourcemanager.MockNodes;
+//import org.apache.hadoop.yarn.server.resourcemanager.rmnode.RMNode;
+//import org.apache.hadoop.yarn.server.resourcemanager.scheduler.event.NodeAddedSchedulerEvent;
+//import org.apache.hadoop.yarn.server.resourcemanager.scheduler.event.NodeRemovedSchedulerEvent;
+//import org.apache.hadoop.yarn.util.resource.Resources;
 //import org.junit.Test;
 
```

Comment:
Actually, @klucar , can you please remove the whole ```TestMyriadScheduler``` class? It's not used anymore. It was meant to test FGS functionality, but I added groovy tests for that.

Also, ```MesosModule``` present in under ```test``` can also be removed, since ```TestMyriadScheduler``` was meant to use the test version of ```MesosModule```.